src/main/java/se/lth/base/server/data/User.java

```diff
@@ -75,6 +78,14 @@ public String getFirstName() {
   public double getRating() {
     return rating;
   }
+  /**
+   * Gets the rating of the user
```

Comment:
Should be sets not gets, right?

```suggestion
   * Sets the rating of the user
```